test/box/fix-memory-corruption-client.lua

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env tarantool
+
+local fiber = require('fiber')
+local net_box = require('net.box')
+
+local fibers= {}
+for i = 1, 1e2 do
```

Comment:
Why the hell couldn't you simply write 100?

Reply:
fixed